Blockzilla/OverlayView.swift

```diff
@@ -146,6 +182,43 @@ class OverlayView: UIView {
                 }
                 self.setAttributedButtonTitle(phrase: query, button: self.searchButton)
                 self.updateCopyConstraint(showCopyButton: showCopyButton)
+                
+                // Update the SaveURL button's position accordingly:
+                self.updateSaveURLConstraint()
+            }
+        }
+    }
+    
+    private func updateSaveURLConstraint() {
+        if currentURL == "" {
+            autoCompleteButton.isHidden = true
+            autoCompleteBorder.isHidden = true
+        } else {
+            autoCompleteButton.isHidden = false
+            autoCompleteBorder.isHidden = false
+            
+            if copyButton.isHidden {
+                if searchButton.isHidden || searchQuery.isEmpty {
+                    // Only visible button
+                    autoCompleteButton.snp.remakeConstraints { make in
+                        make.top.leading.trailing.equalTo(self)
+                        make.height.equalTo(56)
```

Comment:
56 -> is there a UX constant in the code that can be used here? If not please define one